public/Leo Jayachandran/Glossarizer/libretextsGlossarizer.js

```diff
@@ -93,19 +100,23 @@ class LibreTextsGlossarizer {
                 }
                 break;
             case "ichem":
+                localStorage.setItem("glossarizerType", "ichem");
                 let ichemPage = 278612;
                 retrievedGlossary = await getGlossaryJSON(ichemPage);
                 break;
             case "achem":
+                localStorage.setItem("glossarizerType", "achem");
                 let achemPage = 278614;
                 retrievedGlossary = await getGlossaryJSON(achemPage);
                 break;
             case "ochem":
+                localStorage.setItem("glossarizerType", "ochem");
                 let ochemPage = 278613;
                 retrievedGlossary = await getGlossaryJSON(ochemPage);
                 break;
             case "textbook": //The textbook should be the default option
             default:
+                localStorage.setItem("glossarizerType", "textbook");
                 const coverPage = await LibreTexts.getCoverpage();
                 const subdomain = window.location.origin.split('/')[2].split('.')[0];
```

Comment:
No need to replace this here, but I just wanted to make sure that we do have the function 
`const [subdomain, path] = LibreTexts.parseURL(url = window.location.href);`
if you are doing this operation often.

Reply:
I wouldn't imagine this operation being done that often unless the user decides to constantly jump between glossaries.